shared/build.gradle.kts

```diff
@@ -25,7 +25,7 @@ kotlin {
         summary = "Some description for the Shared Module"
         homepage = "Link to the Shared Module homepage"
         version = "1.0"
-        ios.deploymentTarget = "14.1"
+        ios.deploymentTarget = "14.0"
```

Comment:
How about the podfile of iOS target? It always sets the minimum version to 11.

Reply:
Sorry don't get what you mean here 🙏

Reply:
When you update it here but leave the min ios version in podfile untouched, the application can't be built